ci/cron/wednesday.yml

```
@@ -16,54 +16,97 @@ jobs:
   timeoutInMinutes: 60
   pool:
-    name: linux-pool
-    demands: assignment -equals default
+    vmImage: ubuntu-latest
   steps:
   - checkout: self
     persistCredentials: true
-  - bash: ci/dev-env-install.sh
   - bash: |
-      set -euo pipefail
-      eval "$(./dev-env/bin/dade-assist)"
+      set -euxo pipefail
 
-      setvar() {
-          echo "Setting '$1' to '$2'"
-          echo "##vso[task.setvariable variable=$1;isOutput=true]$2"
+      if header=$(git config 'http.https://github.com/digital-asset/daml.extraheader'); then
+          AUTH="$header"
+      else
+          AUTH="Authorization: basic $(git config remote.origin.url | grep -o '://.*:.*@' | cut -c4- | rev | cut -c2- | rev)"
+      fi
+
+      BASE_SHA=$(git rev-parse HEAD)
+      az extension add --name azure-devops
+      echo "$(System.AccessToken)" | az devops login --org "https://dev.azure.com/digitalasset"
+
+      reset() {
+          git checkout -f $BASE_SHA
+          git reset --hard
       }
+      open_pr() {
+          local branch title body out pr_number
+          branch=$1
+          title="$2"
+          body="$3"
+          out=$4
+          git branch -D $branch || true
+          git checkout -b $branch
+          git add .
+          git -c user.name="Azure Pipelines DAML Build" \
+              -c user.email="support@digitalasset.com" \
+              commit \
+              -m "$(printf "$title\n\n$body\n\nCHANGELOG_BEGIN\nCHANGELOG_END\n")"
+          git push origin $branch:$branch
+          pr_number=$(curl -H "Content-Type: application/json" \
+                           -H "$AUTH" \
+                           --silent \
+                           --location \
+                           -d "{\"title\": \"$title\", \"head\": \"$branch\", \"base\": \"master\", \"body\": \"$body\"}" \
+                           https://api.github.com/repos/digital-asset/daml/pulls \
+                      | jq '.number')
+          az pipelines build queue \
+              --branch $branch \
+              --definition-name "digital-asset.daml" \
+              --org "https://dev.azure.com/digitalasset" \
+              --project daml
+          echo $pr_number > $out
+      }
+      assign_and_label() {
+          local pr_number, assignee
+          pr_number=$1
+          assignee=$2
+          curl -H "Content-Type: application/json" \
+               -X PATCH \
+               -u $AUTH \
+               --silent \
+               --location \
+               -d "{\"assignees\": [\"$assignee\"], \"labels\": [\"Standard-Change\"]}" \
+               https://api.github.com/repos/digital-asset/daml/issues/$pr_number
+          # For the purposes of "common" features, such as assignees and
+          # labels, PRs are issues as far as the GH API is concerned.
+      }
+      rotate() {
+          local tmp
+          tmp=$(mktemp)
+          cp release/rotation $tmp
+          cat <(tail -n +2 $tmp) <(head -1 $tmp) > release/rotation
```

You don't need to get `cat` involved here.

```suggestion
          (tail -n +2 $tmp; head -1 $tmp) > release/rotation
```

---

You always want to oust my cats. 😿

---

Let the cat sleep. It's had a hard day.